static/js/stat_var_hierarchy/stat_var_hierarchy.tsx

```diff
@@ -283,4 +297,25 @@ export class StatVarHierarchy extends React.Component<
         return _.cloneDeep(resp.data["path"]).reverse();
       });
   }
+
+  private onMouseOverInfoIcon = () => {
+    const html =
+      "<ul><li>The number in parentheses represents the number of available" +
+      "stat vars within the group that we have for the chosen place(s).</li>" +
+      "<li>Greyed out stat var groups have no available stat vars for the " +
+      "chosen place(s), but can still be expanded for you to explore.</li></ul>";
+    const topOffset = 30;
```

Comment:
moving the const to the top of the file?

Reply:
done